supabase/functions/transcribe-audio/index.ts

```diff
@@ -24,30 +24,19 @@ const processAudio = async (req: Request) => {
   const arrayBuffer = await req.arrayBuffer();
 
   let transcript: string;
-  let embeddings: any;
   try {
     const filenameTimestamp = `adeus_wav_${Date.now()}.wav`;
     const wavFile = await toFile(arrayBuffer, filenameTimestamp);
 
-    // const { data, error } = await supabase.storage
-    //   .from("test")
-    //   .upload(filenameTimestamp, wavFile);
-
-    // if (error) {
-    //   console.error("Error uploading file:", error);
-    // }
-
     const transcriptResponse = await openaiClient.audio.transcriptions.create({
       file: await toFile(wavFile, filenameTimestamp),
       model: "whisper-1",
       prompt:
-        'If this audio file does not contain any speech, please return "None"',
+        'Listen to the entire audio file, if no audio is detected then respond with "None" ',
```

Comment:
Did you test this out by chance? I'm curious if the "speech" part makes it a bit more specific, where it appears that with your update it wouldn't respond with None if there was noise but no detectable speech.

Reply:
I haven't yet.